### PR TITLE
docs: v1.5.3 changelog

### PR DESCRIPTION
**Pull request by alexmt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+
+### Other
+
+* New Project and Application CRD settings ([#2900](https://github.com/argoproj/argo-cd/issues/2900), [#2873](https://github.com/argoproj/argo-cd/issues/2873)) that allows customizing Argo CD behavior.
+* Upgraded Dex (v2.22.0) enables seamless [SSO integration](https://www.openshift.com/blog/openshift-authentication-integration-with-argocd) with Openshift.
+
+
+#### Enhancements
+
+* feat: added --grpc-web-root-path for CLI.
```
Comment (jannfis) [inline CHANGELOG.md:38]: Missing issue link

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,72 @@
...
+* feat: upgrade dex to 2.22.0 (#3468)
+* feat: support user specified account token ids (#3425)
+* feat: support separate Kustomize version per application (#3414)
+* feat: add support for dex prometheus metrics (#3249)
+* feat: add settings troubleshooting commands to the 'argocd-util' binary (#3398)
+* feat: Let user to define meaningful unique JWT token name (#3388)
+* feat: Display link between OLM ClusterServiceVersion and it's OperatorGroup (#3390)
+* feat: Introduce sync-option SkipDryRunOnMissingResource=true (#2873) (#3247)
+* feat: support normalizing CRD fields that use known built-in K8S types (#3357)
+* feat: Whitelisted namespace resources
```
Comment (jannfis) [inline CHANGELOG.md:50]: Missing issue link

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,72 @@
...
+* fix: 'argocd sync' does not take into account IgnoreExtraneous annotation (#3486)
+* fix: CLI renders flipped diff results (#3480)
+* fix: GetApplicationSyncWindows API should not validate project permissions (#3456)
+* fix: argocd-util kubeconfig should use RawRestConfig to export config (#3447)
+* fix: javascript error on accounts list page (#3453)
+* fix: support both <group>/<kind> as well as <kind> as a resource override key (#3433)
+* fix: Updating to jsonnet v1.15.0 fix issue #3277 (#3431)
+* fix for helm repo add with flag --insecure-skip-server-verification (#3420)
+* fix: app diff --local support for helm repo. #3151 (#3407)
+* fix: Syncing apps incorrectly states "app synced", but this is not true
```
Comment (jannfis) [inline CHANGELOG.md:64]: Missing issue link

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,72 @@
...
+
+* New Project and Application CRD settings ([#2900](https://github.com/argoproj/argo-cd/issues/2900), [#2873](https://github.com/argoproj/argo-cd/issues/2873)) that allows customizing Argo CD behavior.
+* Upgraded Dex (v2.22.0) enables seamless [SSO integration](https://www.openshift.com/blog/openshift-authentication-integration-with-argocd) with Openshift.
+
+
+#### Enhancements
+
+* feat: added --grpc-web-root-path for CLI.
+* feat: limit the maximum number of concurrent login attempts (#3467)
+* feat: upgrade kustomize version to 3.5.4 (#3472)
```
Comment (jannfis) [inline CHANGELOG.md:40]: Maybe the upgrade of bundled Kustomize should also be noted in the `Multiple Kustomize Versions` section above, in a side sentence.

**Review by jannfis — CHANGES_REQUESTED** [review]
Awesome! I have just found a few minor quirks, please see below.

File: CHANGELOG.md (hunk 2 above)
Comment (alexmt) [reply CHANGELOG.md:50]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (alexmt) [reply CHANGELOG.md:64]: fixed

File: CHANGELOG.md (hunk 1 above)
Comment (alexmt) [reply CHANGELOG.md:38]: fixed

**Comment by alexmt** [comment]
Addressed your comments @jannfis . I've also noticed that customize documentation was incomplete and cli flag name had a typo. Fixed in as well. Please take another look

**Review by jannfis — APPROVED** [review]
LGTM

**Pull request merged**